src/stored-atom.ts

```diff
@@ -0,0 +1,29 @@
+import { Atom } from './atom';
+import { Observable } from 'rxjs';
+import { identity } from 'ramda';
+
+interface StoredAtomOptions<T> {
```

Comment:
If you started with `IName` convention for interfaces, let's stick to it or abandon at all.